.gitlab/e2e/k8s_e2e.yml

```diff
@@ -0,0 +1,82 @@
+
+---
+# e2e stage
+# Jobs with the k8s_e2e template
+
+.k8s_e2e_template:
+  stage: k8s_e2e
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/gitlab_agent_deploy$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  dependencies: []
+  variables:
+    LANG: C.UTF-8
+  before_script:
+    - export DOCKER_REGISTRY_LOGIN=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DOCKER_REGISTRY_LOGIN_SSM_KEY)
+    - export DOCKER_REGISTRY_PWD=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DOCKER_REGISTRY_PWD_SSM_KEY)
+    - export DD_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $API_KEY_DDDEV_SSM_NAME)
+
+.k8s-e2e-cws-cspm-init:
+  - set +x
```

Comment:
This is unsafe, can you remove this ?

Reply:
I'd say there should be no functional changes to the content of the jobs in this PR, this should purely be a refactoring PR. This `set +x` was present in the job before it was moved, it should stay here.